pages/Elevator-Plate.md

```diff
@@ -0,0 +1,7 @@
+## Obtaining
+The Elevator Plate can be crafted in an [Enchanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+Elevator plates are automatically connected when sharing the X and Z coordinates.
```

Comment:
You used `Elevator Plate` in the previous and next paragraphs. Then for the sake of consistency please do so in here too.